### PR TITLE
Astro 1/16 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/release-notes.md [hunk 1]
```diff
@@ -21,6 +21,25 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
+
+<HostedBadge/>
+
+:::warning
+
+This feature is in [Public Preview](https://docs.astronomer.io/astro/feature-previews).
+
+:::
+
+You can now configure a network connection between Astro and an AWS VPC without contacting Astronomer support. Astro automatically handles creating a connection request and provides instructions for completing the setup yourself. See [Create a privatge connection between Astro and AWS](https://docs.astronomer.io/astro/connect-aws?tab=VPC%20peering#create-a-private-connection-between-astro-and-aws).
```
Comment (jwitz) [inline astro/release-notes.md:36]:
```suggestion
```

File: astro/release-notes.md [hunk 2]
```diff
@@ -21,6 +21,14 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
...
 
+## January 16, 2023
+
+
+### Additional improvements
+
+- The Cloud UI Deployment analytics page now shows **CPU Usage Per Pod (%)** and **Memory Usage Per Pod (MB)** as a percentage of your total available resources rather than the resources of a single worker Pod, such that these metrics will never show Deployment resources usage as exceeding 100%.
+- The maximum value for worker queue **Max # of workers** has increased from 30 to 100.
+
 ## January 9, 2023
```
Comment (jwitz) [inline astro/release-notes.md:32]:
```suggestion
## January 9, 2024
```
Facepalm @ myself

File: astro/release-notes.md [hunk 3]
```diff
@@ -21,6 +21,14 @@ Astronomer is committed to continuous delivery of both features and bug fixes to
 
 <!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->
 
+## January 16, 2023
+
```
Comment (jwitz) [inline astro/release-notes.md:25]:
```suggestion
## January 16, 2024
```

**Review by lzdanski — APPROVED** [review]
After you merge the date updates looks like you're good to go!

**Pull request merged**